Jack.Tests/paket.references

```diff
@@ -1,3 +1,5 @@
 xunit.runner.console
 xunit.core
 Unquote
+
+FSharpx.Collections
```

Comment:
I'm separating (with a blank-line) the references required for test-discovery and assertions from the references required by the actual test code.